docs/source/user-guide/concepts/index.rst

```diff
@@ -10,6 +10,7 @@ Explore the links to learn more about conda foundations.
    conda-commands
    packages
    pkg-specs
+   pkg-search-install-spec
```

Comment:
Could we just name this "search" since this will show up in the URL of the docs?

Reply:
Yeah, good idea. I struggled with what to name the file for a while. :)